entrypoint.sh

```diff
@@ -0,0 +1,14 @@
+set -e
+
+echo "#################################################"
+echo "Changing directory to 'BUILD_DIR' $BUILD_DIR ..."
+# cd $BUILD_DIR
+ls -a
+cd $BUILD_DIR
+
+echo "#################################################"
+echo "Now deploying to GitHub Pages..."
+
+npm install && \
+npm run build
+# npm run deploy
```

Comment:
Need to add the git part of the deploy but there is an error with some of the testing files on the build
```
{ Error: Error in file: /github/workspace/root_directory/website/static/oss_logo.png

spawn /github/workspace/root_directory/website/node_modules/optipng-bin/vendor/optipng ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:246:19)
    at onErrorNT (internal/child_process.js:427:16)
    at processTicksAndRejections (internal/process/next_tick.js:76:17)
  errno: 'ENOENT',
  code: 'ENOENT',
  syscall:
   'spawn /github/workspace/root_directory/website/node_modules/optipng-bin/vendor/optipng',
  path:
   '/github/workspace/root_directory/website/node_modules/optipng-bin/vendor/optipng',
  spawnargs:
   [ '-strip',
     'all',
     '-clobber',
     '-fix',
     '-o',
     3,
     '-out',
     '/tmp/876cce92-cd75-4bd0-8457-cc70862274d5',
     '/tmp/7fadaff6-7311-4255-ad60-6635ef45db78' ],
  killed: false,
  stdout: '',
  stderr: '',
  failed: true,
  signal: null,
  cmd:
   '/github/workspace/root_directory/website/node_modules/optipng-bin/vendor/optipng -strip all -clobber -fix -o 3 -out /tmp/876cce92-cd75-4bd0-8457-cc70862274d5 /tmp/7fadaff6-7311-4255-ad60-6635ef45db78',
  timedOut: false }
```